Tests/UI/New UI/PaymentFormFactoryTests.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  PaymentFormFactoryTests.swift
+//  
+//
+//  Copyright © 2022 Checkout. All rights reserved.
+//
+
+import XCTest
+@testable import Frames
+
+class PaymentFormFactoryTests: XCTestCase {
+
+  func testGetPaymentFormViewController() {
+    let viewController = PaymentFormFactory.getPaymentFormViewController(
+      billingFormData: nil,
+      paymentFormStyle: nil,
+      billingFormStyle: nil)
+    XCTAssertNotNil(viewController)
```

Comment:
Could we also dig into the properties of the controller to check that the visible properties are as expected?

Maybe also add tests for those properties not being nil and seeing the factory generates the VC with all the properties you would expect for the inputs?

Reply:
Had a look and the ViewModel inside VC is internal for reading, so you could check things like:
- viewController.viewModel. cardValidator
- viewController.viewModel. billingFormData
- viewController.viewModel. paymentForStyle
- viewController.viewModel. billingFormStyle